libs/database/src/lib/services/authentication.service.v2.ts

```diff
@@ -307,20 +302,16 @@ export class AuthenticationServiceV2 {
     account.isGuest = false;
     account.email = dto.email;
 
-    if (Environment.PDPA_ACCEPT_DATE) {
-      const ofPDPA = Environment.PDPA_ACCEPT_DATE.split(',');
+    const datePDPA = Environment.PDPA_ACCEPT_DATES;
```

Comment:
```suggestion
    if (Environment.PDPA_ACCEPT_DATES.length) {
      (account.pdpa ??= {})[Environment.PDPA_ACCEPT_DATES[0]] = true;
    }
```

ควรจะเซ็ท accept แค่ตัวล่าสุดนะครับ